Palette (Pie Chart): update demo description (#2239)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/Palette/description.md

> The PieChart, like other DevExtreme Data Visualization components, allows you to apply several predefined palettes. In this demo, you can use two drop-down menus under the PieChart to choose a palette and change the way how the palette is extended when it has an insufficient number of colors.

Reviewer: - "several" - we use "multiple".
- "when it has an insufficient number of colors" - insufficient for what?
- "is extended" - avoidable passive voice.

```suggestion
The PieChart, like other DevExtreme Data Visualization components, allows you to apply several predefined palettes. In this demo, you can use two drop-down menus under the PieChart to choose a palette and change the way to extend it when its number of colors is insufficient to paint each series point differently.
```

> ## Configure Palette

Reviewer: If you agree to divide the section in two (see below), then we need a different heading.

```suggestion
## Specify a Palette
```

> A palette is a set of colors that mix well with each other. To apply the needed color scheme you can assign it to a [palette](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#palette) property. It accepts either the name of a [predefined palette](/Documentation/Guide/Themes_and_Styles/SVG-Based_Components_Customization/#Palettes) or an array of colors. In this demo, you can use the "Palette" selector to apply a new set of colors.

Reviewer: ```suggestion
A palette is a set of colors that mix well with each other. To apply the needed color scheme, you can assign it to the [palette](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#palette) property. It accepts either the name of a [predefined palette](/Documentation/Guide/Themes_and_Styles/SVG-Based_Components_Customization/#Palettes) or an array of colors. In this demo, you can use the "Palette" selector to apply a new set of colors.
```

> When the number of palette colors is less than the number of points in a series, you can use the [paletteExtensionMode](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#paletteExtensionMode) property to specify what to do with colors in the palette. This property can accept one of the following values:

Reviewer: - There is enough information about `paletteExtensionMode` to make a separate section of it.
- "in a series" - avoidable modifier.
- "what to do with colors" - the wording doesn't seem to belong to documentation.

```suggestion
## Extend a Pallete

When the number of palette colors is less than the number of series points, you can use the [paletteExtensionMode](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#paletteExtensionMode) property to specify how to extend the palette. This property can accept one of the following values:
```

> - "blend"

Reviewer: We usually display string values in italics. Also, the list item description is displayed on the same line as the list item itself. Don't forget to add at least four space characters after each list item to make a line break. Please edit other list items similarly.

```suggestion
- *"blend"*        
```

> The PieChart, like other DevExtreme Data Visualization components, allows you to apply multiple predefined palettes. In this demo, you can use two drop-down menus under the PieChart to choose a palette and change the way to extend it when its number of colors is insufficient to paint each series point differently.

Reviewer 2: ```suggestion
The PieChart, like other DevExtreme Data Visualization components, allows you to apply multiple predefined palettes. In this demo, you can use two drop-down menus under the PieChart to choose a palette and change the way it is extended when the number of colors is insufficient to paint each series point differently.
```

## Specify a Palette

> A palette is a set of colors that mix well with each other. To apply the needed color scheme, you can assign it to the [palette](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#palette) property. It accepts either the name of a [predefined palette](/Documentation/Guide/Themes_and_Styles/SVG-Based_Components_Customization/#Palettes) or an array of colors. In this demo, you can use the "Palette" selector to apply a new set of colors.

Reviewer 2: `that mix well with each other` I'm not sure this is an accurate description of a palette since it is more about the range of colors, perhaps even complementing colors

Author: In this case, a palette is the range of colors to be applied to the series points
